### DISCUSSION TITLE
New category for text and media alerts for streaming

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#7] [Ideas]
The "news" category is somewhat short for channels that broadcast certain events as a notification inside channel. 

Examples would be: TV programs to be broadcasted in the guide (TV Guide), sporting events of the last week (fixture), recommendations for natural disasters, and so on. The user can even choose an interactive way like BBC Red Button, a teleservice to find out about an event on this network.

I think we should have a new category for this.

Based on iptv-org/iptv-org.github.io#470 (teletext issue), iptv-org/iptv-org.github.io#238 (newscast category) and https://github.com/iptv-org/database/pull/159 (PR).

**Comment by freearhey** [comment]
Were you able to come up with a name for the new category and definition of it?

**Comment by sguinetti** [comment]
A better word to define this would be “interactive”. Unlike other categories, the transmission is not unidirectional but bidirectional, in which the user “interacts” with it by means of a click, a button, and so on.

When there is “interaction”, then there is user participation. This user chooses how to display alerts from their neighborhood or access on-demand content from the stream platform. I'm fine with “interactive” being the inclusive term.

**Comment by freearhey** [comment]
Added in https://github.com/iptv-org/database/pull/19136
